router/core/graphql_prehandler.go

```diff
@@ -357,6 +358,24 @@ func (h *PreHandler) Handler(next http.Handler) http.Handler {
 
 func (h *PreHandler) handleOperation(req *http.Request, buf *bytes.Buffer, httpOperation *httpOperation) (*operationContext, error) {
 
+	// Handle the case when operation information are provided as GET parameters
+	if req.Method == http.MethodGet {
+		q := req.URL.Query()
+		req := GraphQLRequest{
+			OperationName: q.Get("operationName"),
+			Query:         q.Get("query"),
+			Variables:     []byte(q.Get("variables")),
+			Extensions:    []byte(q.Get("extensions")),
```

Comment:
Is it correct that other solutions encode variables & extensions as JSON in the URL query?

Reply:
Yes, https://graphql.github.io/graphql-over-http/draft/#sec-GET